proto/penumbra/penumbra/core/component/auction/v1alpha1/auction.proto

```diff
@@ -173,3 +173,30 @@ message ActionDutchAuctionWithdrawView {
   // reserves commitment.
   repeated asset.v1.ValueView reserves = 2;
 }
+
+message EventDutchAuctionScheduled {
+  AuctionId auction_id = 1;
+  DutchAuctionDescription description = 2;
+}
+
+message EventDutchAuctionUpdated {
+  AuctionId auction_id = 1;
+  DutchAuctionState state = 2;
+}
+
+message EventDutchAuctionEnded {
+  // The reason the auction ended.
+  enum Reason {
+    REASON_UNSPECIFIED = 0;
```

Comment:
i see we never create this variant in `crates/core/component/auction/src/event.rs`. should we remove it? we could always add new variants in the future if needed.

Reply:
answered this question myself, via `rg 'enum \w* {' -C 10 proto/penumbra`.

i see this is an existing pattern, so i'll mark this as resolved.

Reply:
https://buf.build/docs/lint/rules#enum_value_prefix

All enums should have a zero value. proto3 doesn't differentiate between set and unset fields, so if an enum field isn't explicitly set, it defaults to the zero value. If an explicit zero value isn't part of the enum definition, this defaults to the actual zero value of the enum. For example, if you had the following .proto file, any Uri with scheme not explicitly set would default to SCHEME_FTP:

Reply:
thanks for the background @hdevalence & @erwanor! i appreciate hearing the underlying reason for these `FOO_UNSPECIFIED` variants.